aspnetcore/performance/caching/response.md

```diff
@@ -35,6 +35,9 @@ The web server can cache responses by adding the response caching middleware. Se
 
 The `ResponseCacheAttribute` specifies the parameters necessary for setting appropriate headers in response caching. See [ResponseCacheAttribute](https://docs.microsoft.com/en-us/aspnet/core/api/microsoft.aspnetcore.mvc.responsecacheattribute)  for a description of the parameters.
 
+>[!WARNING]
+> Remember to disable caching for content that may contain information for authenticated clients. Caching should only be used for content available to anyone.
```

Comment:
~~Remember to~~ disable caching for content that may contain information for authenticated clients.